util/DEPS.py

```diff
@@ -0,0 +1,24 @@
+# If you're familiar with Chromium or other Google projects:
+#   Sorry, this file actually has nothing to do with gclient.
+#   I just borrowed the basic concept and naming scheme.
+
+deps = {
+    'ACT': {
+        'url': 'https://github.com/EQAditu/AdvancedCombatTracker/releases/download/3.4.5.265/ACTv3.zip',
```

Comment:
I think this needs to be 3.4.5.266 because that's the version the the 2.0.5.0 version of the FFXIV ACT plugin depends on.  Without bumping that, I get an `MSB3277: Found conflicts between different versions of "Advanced Combat Tracker" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.` warning in my log when building.

Reply:
This stuff was copied from OverlayPlugin and I'm building against an older version of the FFXIV plugin (2.0.4.0). I think Akurosia bumped the plugin version and didn't touch the ACT version.

Reply:
BTW here's the message from back then and why I haven't updated the FFXIV plugin, yet: https://github.com/ngld/OverlayPlugin/issues/43

Apparently that's no longer the case, though. I'll see if I can figure out what the current versions for CN players are since I'd expect them to run into the same issue.

Reply:
Mmm, that's interesting.  I am quite surprised that no Chinese players have complained about Cactbot dependencies, then.

Reply:
I think I will land this and then bump the ffxiv plugin version back to 2.0.4.0 to match OverlayPlugin.